corehq/apps/app_manager/detail_screen.py

```diff
@@ -376,7 +377,7 @@ def _xpath_template(self, type):
         if type == 'sort':
             return "if({xpath} = '{key}', {i}, "
         if type == 'display':
-            return "if({xpath} = '{key}', ${key_as_var}, "
+            return "if(selected({xpath}, '{key}'), ${key_as_var}, ''), "
```

Comment:
Thinking about sorting some more, I think it may be better to update the expression for both display and sort columns, for the sake of keeping the code simpler, even though it's possible users won't **want** to sort by a multi-value property.

Can you clarify, if you were to update the sort calculation to use `selected`:
- Would that change the behavior of sorting a single-value property?
- What exactly would the sorting behavior be for a multi-value property?

Reply:
After changing the sort calculation:

- Single value sorting seems to still work as intended
- The sorting on a multi value property is dependent on the order of the answers in the multi-select question

If the answers appear like this in the question:
- [ ]  Option 1
- [ ] Option 3
- [ ] Option 2

Then responses with `Option 1` will appear first.
Followed by responses with `Option 3`.
Then responses with `Option 2`.

Reply:
Interesting. That behavior seems okay to me, so long as it's documented. Let's use the new calculation for both sorting and display.